src/now.rs

```diff
@@ -22,6 +22,14 @@ pub(crate) use self::sys::*;
 )))]
 mod sys {
     pub(crate) fn system_time() -> std::time::SystemTime {
+        #[cfg(all(
+            not(feature = "js"),
+            any(target_arch = "wasm32", target_arch = "wasm64"),
```

Comment:
This is what `target_family = "wasm"` exists for. Unless you want to tailor the mention of wasm32 in the panic message 🤔

Reply:
On second thought, I don’t even know if wasm-bindgen supports wasm64

Reply:
I would have used that, but the `cfg` on `mod sys` does this as well

Reply:
> On second thought, I don’t even know if wasm-bindgen supports wasm64

It does not: https://github.com/rustwasm/wasm-bindgen/issues/4436

Reply:
Yeah I think just sticking to what the existing `cfg`s are is fine here. We can improve those later if there's a reason to.